wiki/Tournaments/SOLMT/2019/en.md

```diff
@@ -0,0 +1,373 @@
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2019-10-14/2019-11-6 |
+| Qualifiers | 2019-11-23/2019-11-24 |
+| Group Stage First Week| 2019-11-30/2019-12-01 |
+| Group Stage Second Week| 2019-12-07/2019-12-08 |
+| Round of 32 | 2019-12-14/2019-12-15 |
+| Round of 16 | 2019-12-21/2019-12-22 |
+| Quarterfinals | 2019-12-27/2019-12-28 |
+| Semifinals | 2020-01-4/2020-01-5 |
+| Finals | 2020-01-11/2020-01-12 |
+| Grand Finals | 2020-01-18/2020-01-19 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Razer Huntsman Elite (Red Switch), Tournament Badge, 3 Months of osu!supporter tag |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | Corsair K70 RapidFire MK.2 (MXCherry Speed Switch), 2 Months of osu!supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | Razer Huntsman Tournament Edition TKL (Red Switch), 1 Month of osu!supporter tag  |
+![SOLMT Badge](img/badge.png "SoL Winner Badge")
+
+## Organisation
+
+The Speed of Light Mania Tournament is run by various community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Manager |![][flag_MX] [[Crz]Crystalize](https://osu.ppy.sh/users/3072763), ![][flag_US] [ZeroRam](https://osu.ppy.sh/users/8454732), ![][flag_MX] [Candy](https://osu.ppy.sh/users/3964918), ![][flag_MX] [demiantorp](https://osu.ppy.sh/users/8291099)|
+| Map Selector |![][flag_MX] [[Crz]Crystalize](https://osu.ppy.sh/users/3072763), ![][flag_US] [-Lethal-](https://osu.ppy.sh/u/5125822), ![][flag_US] [Dergo](https://osu.ppy.sh/users/7439226), ![][flag_US] [Hubert](https://osu.ppy.sh/users/6430626), ![][flag_US] [Penguinosity](https://osu.ppy.sh/users/10235296)|
+| Streamer | ![][flag_US] [-Lethal-](https://osu.ppy.sh/u/5125822), ![][flag_US] [Hubert](https://osu.ppy.sh/users/6430626), ![][flag_US] [ZeroRam](https://osu.ppy.sh/users/8454732)
+| Commentator | ![][flag_US] [Penguinosity](https://osu.ppy.sh/users/10235296) ![][flag_ID] [[Crz]Crysarlene](https://osu.ppy.sh/users/5492871), ![][flag_MX] [demiantorp](https://osu.ppy.sh/users/8291099), ![][flag_MX] [[Toni]](https://osu.ppy.sh/users/9260169), ![][flag_MX] [KanekiKen-kun](https://osu.ppy.sh/users/12157233), ![][flag_US] [Mango Juul Pod](https://osu.ppy.sh/users/11542560)|
+| Designer | ![][flag_MX][Candy](https://osu.ppy.sh/users/3964918)
+| Referee | ![][flag_US] [Hubert](https://osu.ppy.sh/users/6430626), ![][flag_US] [-Lethal-](https://osu.ppy.sh/u/5125822), ![][flag_US] [Mango Juul Pod](https://osu.ppy.sh/users/11542560), ![][flag_MX] [urielbg](https://osu.ppy.sh/users/4598040), ![][flag_SG] [Rycy](https://osu.ppy.sh/users/7394522), ![][flag_PE] [Tunning23](https://osu.ppy.sh/users/9396363)
+| Statistician | ![][flag_MX] [[Crz]Crystalize](https://osu.ppy.sh/users/3072763), ![][flag_US] [ZeroRam](https://osu.ppy.sh/users/8454732) |
```

Comment:
`[Crz]Crystalize` -> `\[Crz\]Crystalize`

Add spaces after `|` and content where needed (L43-L44)